packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/edit-connector/edit-connector.html

```diff
@@ -0,0 +1,91 @@
+        <span class="oui-paragraph">
+            Version: {{$ctrl.availableConnector.version}} | Author:
+            {{$ctrl.availableConnector.author}} |
+            <a
+                ng-href="{{$ctrl.availableConnector.documentationUrl}}"
+                target="_blank"
+                rel="noopener"
+                >docs</a
+            >
+        </span>
+    </div>
+    <oui-button variant="primary" size="l" ng-click="$ctrl.updateConnector()">
+        <span data-translate="pci_databases_connectors_edit_button"></span>
+    </oui-button>
+</div>
+
+<oui-tabs>
+    <oui-tabs-item
+        ng-repeat="group in $ctrl.availableConnector.configuration.getGroups()"
+        heading="{{group.name}}"
+        subheading="{{group.required ? 'required' : ''}}"
+        variant="promotion"
+    >
+        <div class="row">
+            <div class="col-md-6">
+                <div
+                    ng-if="group.name === 'Transforms'"
+                    class="oui-message oui-message_info"
+                    role="alert"
+                >
+                    <span
+                        class="oui-message__icon oui-icon oui-icon-info-circle"
+                        aria-hidden="true"
+                    ></span>
+                    <p class="oui-message__body">
+                        Transformations enable simple message-at-a-time
+                        modifications. See
+                        <a
+                            href="https://kafka.apache.org/documentation.html#connect_transforms"
+                            target="_blank"
+                            rel="noopener"
+                            >documentation</a
+                        >
+                        for more details.
+                    </p>
+                </div>
+                <div
+                    ng-if="group.name === 'Extra'"
+                    class="oui-message oui-message_info"
+                    role="alert"
+                >
+                    <span
+                        class="oui-message__icon oui-icon oui-icon-info-circle"
+                        aria-hidden="true"
+                    ></span>
+                    <p class="oui-message__body">
+                        Here you can define connector configurations parameters,
+                        that are not defined in the configuration schema.
+                    </p>
+                </div>
+                <form name="connector-{{$index}}">
+                    <connector-input
+                        ng-repeat="field in $ctrl.availableConnector.configuration.getFields(group.name)"
+                        data="field"
+                        model="$ctrl.model"
+                        configuration="$ctrl.availableConnector.configuration"
+                    >
+                    </connector-input>
+                </form>
+            </div>
+            <div class="col-md-6 d-none d-md-block">
```

Comment:
**question:** Why using `d-none` helper? Do we have to keep this element?

Reply:
We use `d-none` here to hide the element on lower resolution devices, but we do need it on higher resolutions.